apps/src/templates/Notification.jsx

```diff
@@ -119,11 +129,23 @@ class Notification extends Component {
     dismissible: PropTypes.bool.isRequired,
     onDismiss: PropTypes.func,
     newWindow: PropTypes.bool,
+    // analyticId is only posted when a primary button is provided.
+    // It's not used by the array of buttons.
     analyticId: PropTypes.string,
+    responsiveSize: PropTypes.oneOf(['lg', 'md', 'sm', 'xs']),
     isRtl: PropTypes.bool.isRequired,
     onButtonClick: PropTypes.func,
     buttonClassName: PropTypes.string,
 
+    // Optionally can provide an array of buttons.
+    buttons: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
I think I'm a little confused about this array of buttons, because for the first button you can still pass all the button props (`buttonText` etc individually), but this is also allows you to pass an array of buttons. Are we ever anticipating having more than 2 buttons? If not, I'd suggest as future cleanup, either pass props individually for the second button (`secondButtonText`) similar to how we pass them for the first  OR always pass a buttons array and never pass button related props individually.

Reply:
Yes, this is a little goofy because I wanted to maintain full compatibility with the existing uses.  Basically we show the first button if it's available, then we show the array of buttons if they're available.  In reality, I expect one or the other, and you're right that a better long term plan is to only support the array.  (Or maybe support either a single hash or an array of hashes in the same parameter, though not sure if that's a good pattern or not.)  Agreed that this should be a (near-) future cleanup.